### DISCUSSION TITLE
AKS Edge Essentials multi-node - is it possible to connect my Azure Cloud Shell kubectl config to the cluster?

### DISCUSSION THREAD

**Discussion by colinrippeyfinarne** [microsoft/azure_arc#1915] [Q&A] [answered]
HI,

I've deployed the AKS Edge Essentials multi-node scenario.

I can happily RDP onto the AKS-EE-Full-VM, and from there RDP once more onto Node1 and run kubectl commands.

If I pivot out to an Azure Cloud Shell session I can "see" the cluster if I run:

az connectedk8s list -o table 

But from my azure cloud shell I don't know how to configure my ~/.kube/config file to have the correct values to allow me to then switch my kubectl context to the Edge Essentials cluster?

Is this possible?

Colin

**Comment by liorkamrat** [comment]
@lanicolas

**Comment by lanicolas** [comment, marked as answer]
hi @colinrippeyfinarne 

Thank you for using jumpstart. The cluster is on a private network within the nested VM so the K8s is not publicly exposed. You could however use something like this https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/cluster-connect?tabs=azure-cli%2Cagent-version to access it. But it has not been enabled as part of the jumpstart scenario. 

Please let me know if this makes things clearer

**Reply by colinrippeyfinarne** [reply]
Thanks I'll try that.
